test/controllers/auth/AuthActionSpec.scala

```diff
@@ -101,30 +101,26 @@ class AuthActionSpec extends FreeSpec with MustMatchers with MockitoSugar with O
     }
   }
 
+  implicit class Ops[A](a: A) {
+    def ~[B](b: B): A ~ B = new ~(a, b)
+  }
+
   "A user with nino and no SA enrolment must" - {
     "create an authenticated request" in {
 
       val nino = Fixtures.fakeNino.nino
       val retrievalResult: Future[
         Option[String] ~ Enrolments ~ Option[Credentials] ~ ConfidenceLevel ~ Option[UserName] ~ LoginTimes ~ Option[
-          TrustedHelper]] =
+          TrustedHelper] ~ Option[String]] =
         Future.successful(
-          new ~(
-            new ~(
-              new ~(
-                new ~(
-                  new ~(new ~(Some(nino), Enrolments(Set.empty)), Some(Credentials("foo", "bar"))),
-                  ConfidenceLevel.L200),
-                None),
-              LoginTimes(DateTime.now(), None)
-            ),
-            None
-          ))
+          Some(nino) ~ Enrolments(Set.empty) ~ Some(Credentials("foo", "bar")) ~ ConfidenceLevel.L200 ~ None ~ LoginTimes(
```

Comment:
A lot of the retrieval values are used multiple times. Pull them out into variables and reuse them